erpnext/payroll/doctype/overtime_slip/overtime_slip.json

```diff
@@ -0,0 +1,196 @@
+{
+ "actions": [],
+ "autoname": "HR-OVR-SLIP-.#####",
```

Comment:
```suggestion
 "autoname": "HR-OT-SLIP-.#####",
```
Better naming series since OT is short for Overtime?

Reply:
Done